src/pages/Verifier/CheckedContractsView/CheckedContract/ChainAddressForm/index.tsx

```
@@ -81,21 +66,14 @@ const ChainAddressForm = ({
     }
 
-    checkAllByAddresses(
-      checksummedAddress,
-      sourcifyChains.map((c) => c.chainId.toString()).join(",")
-    ).then((res) => {
+    checkAllByAddresses(checksummedAddress, sourcifyChains.map((c) => c.chainId.toString()).join(",")).then((res) => {
       // checkAllByAddresses inputs and outptus multiple addresses.
-      const currentAddressMatches = res.find(
-        (match) => (match.address = checksummedAddress)
-      );
+      const currentAddressMatches = res.find((match) => (match.address = checksummedAddress));
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)